src/dotnet/UI.Blazor.App/Pages/Landing/DownloadButton.razor

```diff
@@ -1,8 +1,15 @@
-<button class="download-btn" data-href="@(Links.Apps.Android)">
-    <div class="text-04 text-headline-1">
-        Download for Android
-    </div>
-    <div>
-        <img draggable="false" src="/dist/images/landing/android.svg" alt="android">
+<button class="download-btn @Class">
```

Comment:
I'd recommend not to change 'Download' button to make 'Back to Main page' button on LandingDownloadLinks.razor.
You can either create a new component or just inline it on LandingDownloadLinks.
Use extra css class to bind scroll behviour in type script.